valve-monitor/web-app/README.md

```diff
@@ -0,0 +1,366 @@
+# Valve Monitor Web App
+
+The Valve Monitor project’s web application allows you to view flow rates, open and
+close solenoid valves, and manage environment variables in a browser.
+
+![The Valve Monitor project’s web application](../images/web-app.png)
+
+To get started, make sure you have a copy of this project’s repository locally, open
+up the `valve-monitor/web-app` folder in your text editor or IDE, and then complete
+the following steps.
+
+- [Dependencies](#dependencies)
+  - [(Recommended) Visual Studio Code Dev Container](#recommended-visual-studio-code-dev-container)
+  - [(Not Recommended) Dependencies without VS Code](#not-recommended-dependencies-without-vs-code)
+- [Configuration (Environment Variables)](#configuration-environment-variables)
+  - [HUB_AUTH_TOKEN](#hub_auth_token)
+  - [HUB_PROJECT_UID](#hub_project_uid)
```

Comment:
We have `HUB_FLEET_UID` as a config also, but it looks like it's never used in the project. I suggest removing from `Config.ts` and `.env.example` if that's the case.